src/Core/Checkout/Order/Aggregate/OrderTransaction/OrderTransactionExtension.php

```diff
@@ -41,12 +44,17 @@ public function getDefinitionClass(): string
 
     public function extendFields(FieldCollection $collection): void
     {
-        $collection->add(
-            new OneToManyAssociationField(
-                'adyenPaymentResponses',
-                PaymentResponseEntityDefinition::class,
-                'order_transaction_id'
-            )
+        $field = new OneToManyAssociationField(
+            'adyenPaymentResponses',
+            PaymentResponseEntityDefinition::class,
+            'order_transaction_id'
         );
+
+        // Ensure the data is not available via the Store API in older Shopware versions.
```

Comment:
@peterojo As you still support older versions, I added the readrpotect field here as otherwise the data could be retrieved using the store API in versions prior to 6.3.5.1

Reply:
@AndreasA this extension does not have the `ApiAware()` flag, so why is it necessary to block older versions?

Reply:
@peterojo 
older versions of Shopware use inverted flags behavior. ReadProtected will be removed in future versions altogether.

- up to 6.3.5.0: It is necessary to specify which fields are not available via the Store api using the ReadProtected flag
- after 6.3.5.1 (probably wanted to do this in 6.3.5.0 already) and newer: It is necessary to mark all fields that should be available via Store API by using the ApiAware flag.

So to ensure older versions of Shopware cannot access the field using Store API, the ReadProtected flag has to be added.